CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 * [Doc] Document --persist-replace in API section (#539)
 * [Fix] Fixed CI issue by updating `invalid_connection_string_duckdb` in `test_magic.py` (#631)
 * [Fix] Refactored `ResultSet` to lazy loading (#470)
-* [Fix] Removed `WITH` when a snippet does not have a dependency (#657)
```

Comment:
Why this line was removed?

Reply:
This got removed by mistake while editing, added back the log.